src/lib/pmd/PmdSupport.ts

```diff
@@ -23,6 +23,7 @@ export abstract class PmdSupport {
     
     // Include custom rule paths into classpath
     const rulePathEntries = await this.getRulePathEntries();
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     rulePathEntries.forEach((pathEntries, language) => {
```

Comment:
Try removing the `language` parameter from the callback altogether. If that doesn't work, try `Array.from(rulePathEntries.values()).forEach((pathEntries) => {`